services/indexes/pvm/writer.go

```diff
@@ -397,29 +405,49 @@ func (w *Writer) indexBlock(ctx services.ConsumerCtx, proposerblockBytes []byte)
 	errs := wrappers.Errs{}
 
 	switch blk := pblock.(type) {
-	case *platformvm.ProposalBlock:
+	case *blocks.ApricotProposalBlock:
 		errs.Add(
 			initializeTx(ver, w.codec, blk.Tx),
```

Comment:
If you use `blocks.Parse` as suggested above, you can delete `initializeTx`.

Reply:
same question on blocks vs block